src/components/dashboard/todos-by-goal/goal-list.tsx

```
@@ -0,0 +1,27 @@
+interface GoalListProps {
+  goals: GoalResponse[];
+}
+
+export default function GoalList({ goals }: GoalListProps) {
+  return (
+    <>
+      <ul className="flex flex-col gap-16">
+        {goals &&
+          goals.map((goal) => (
+            <GoalItem key={`dashboard-goal-${goal.id}`} goal={goal} />
+          ))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)